feat(examples): Add core-utils (#141)
unikraft/catalog: examples/core-utils/README.md

> # Linux Core Utils

Reviewer: ```suggestion
# Linux Coreutils
```

This directory contains a minimal Debian instance featuring ore utils such as ls, cat, pwd, etc., running on Unikraft.

## Set Up

Use `kraft` to run the image and start a Unikraft instance:

> ```bash

Reviewer: ```suggestion
```console
```

kraft run --rm --plat qemu --arch x86_64 -K Kraftfile.{`cmd`} .
```

> Where `cmd` is the command you want to execute.

Reviewer: ```suggestion

Where `cmd` is the command you want to execute.
```

# Linux Coreutils

> This directory contains a minimal Debian instance featuring ore utils such as ls, cat, pwd, etc., running on Unikraft.

Reviewer: ```suggestion
This directory contains a minimal Debian instance featuring core utilities such as `ls`, `cat`, `pwd`, etc., running on Unikraft.
```
